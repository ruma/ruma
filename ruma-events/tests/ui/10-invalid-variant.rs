use ruma_events_macros::AnyEventDeserialize;

#[derive(Clone, Debug, AnyEventDeserialize)]
pub enum AnyRoomEventStub {
    Message(()),
    Redaction(()),
    NotAnEvent(()),
}

fn main() {}
