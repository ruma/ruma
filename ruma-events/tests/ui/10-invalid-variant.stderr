error: found unrecognized enum variant `NotAnEvent`
 --> $DIR/10-invalid-variant.rs:3:24
  |
3 | #[derive(Clone, Debug, AnyEventDeserialize)]
  |                        ^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
